modules/dnn/src/cuda4dnn/primitives/normalize_bbox.hpp

```diff
@@ -111,7 +111,7 @@ namespace cv { namespace dnn { namespace cuda4dnn {
              * or there might be several weights
              * or we don't have to scale
              */
-            if (weight != (T)(1.0))
+            if (weight != static_cast(T)(1.0))
```

Comment:
Wrong `static_cast` syntax, please refer to [cppreference](https://en.cppreference.com/w/cpp/language/static_cast) for more information about how it works.

```cpp
static_cast<T>(1.0)
```